doc: add how to enable JVM debugging for Kafka Connect (#208)
awslabs/eventbridge-kafka-connector: CONTRIBUTING.md

docker-compose -f docker_compose.yaml down --remove-orphans -v
```

##### Remote Debug the Connector

If you need to debug the connector then update the used `docker_compose{_(confluent|redpanda)}.yaml` by uncommenting:
```yaml
services:
  connect:
    image: …
    depends_on: …
    ports:
      - "8083"
      - "5005:5005"
    environment:
      # …
      KAFKA_OPTS: -agentlib:jdwp=transport=dt_socket,server=y,suspend=n,address=*:5005
    volumes:
      - …
```

> The port for JVM debugging is `5005` wich can be changed by changing the mapped host port, e.g. use `9000` instead is `"5005:9000"`. Use your favorite JVM debugging tool and start remote debugging. Do not forget to deploy the connector (see steps above).

Reviewer: ```suggestion
The port for JVM debugging is `5005` wich can be changed by with port mapping configuration e.g., `"5005:9000"` if the desired port is `9000`. Use your favorite JVM debugging tool and start remote debugging. Do not forget to deploy the connector (see steps above).
```

> The port for JVM debugging is `5005` wich can be changed by with port mapping configuration e.g., `"5005:9000"` if the desired port is `9000`. Use your favorite JVM debugging tool and start remote debugging. Do not forget to deploy the connector (see steps above).

Reviewer: Sorry if this was me in my suggestion :(

```suggestion
The port for JVM debugging is `5005` wich can be changed with the shown port mapping configuration e.g., `"5005:9000"` if `address=*:9000` (note that from your local JVM the port would remain `5005` in this example). Use your favorite JVM debugging tool and start remote debugging. Do not forget to deploy the connector (see steps above).
```

Author: > if `address=*:9000` (note that from your local JVM the port would remain `5005` in this example)

Do you mean that it's like `... address=*:9000`? It's the JVM inside Docker.

Reviewer: Sorry, not `JVM` but `JVM debugging tool` - my bad.

Reviewer: Updated suggestion

Author: Okay. Misstake from my side, it should be `"9000:5005"` to switch the port on the host to run the local JVM debugging port at `9000`.
